show-land/sass/_typography.scss

```diff
@@ -0,0 +1,20 @@
+body {
+    font-family: "Lato", sans-serif;
+    font-weight: 400;
+    /* font-size: 16px; */
+    line-height: 1.7;
+    color: $colorWihte;
+    font-size: 95%;
+    }
+    .background-color{
+
+        background-image: linear-gradient(to right ,$colorSecondary,$colorTertiary , $colorPrimary);
+        height: 100vh;
+       
+      }
+      
+      .container{
+      
+        width: 80rem;
+        margin: 0 auto;
+      }
```

Comment:
If you want to have a clean & clear code you can write like this

```
body {
  font-family: "Lato", sans-serif;
  font-weight: 400;
  /* font-size: 16px; */
  line-height: 1.7;
  color: $colorWihte;
  font-size: 95%;
  
  .background-color {
    background-image: linear-gradient(
      to right,
      $colorSecondary,
      $colorTertiary,
      $colorPrimary
    );
    height: 100vh;
  }

  .container {
    width: 80rem;
    margin: 0 auto;
  }
}
```

and for container I am looking at your header, the page has an extra space and you can remove it with this code
```
    width: 95%;
```

or if you like use `rem` you can write like this

```
    width: 75rem;
```

Because `80rem` is too much and creates a gap